libsolidity/analysis/DocStringTagParser.cpp

```diff
@@ -71,7 +71,7 @@ bool DocStringTagParser::visit(VariableDeclaration const& _variable)
 		if (_variable.isPublic())
 			parseDocStrings(_variable, _variable.annotation(), {"dev", "notice", "return", "inheritdoc"}, "public state variables");
 		else
-			parseDocStrings(_variable, _variable.annotation(), {"dev", "inheritdoc"}, "non-public state variables");
+			parseDocStrings(_variable, _variable.annotation(), {"dev", "notice", "inheritdoc"}, "non-public state variables");
```

Comment:
I don't think notice is useful for non-public state variables

Reply:
Should it be removed?

Reply:
@leonardoalt this was mentioned in the chat on thursday. The problem with not allowing `notice` here is:

```solidity
contract Test {
	/// this is an error
	uint y = 1;
	function some() public {
		/// this is not an error
		uint x = 1;
	}
}
```
I find this behaviour weird and would prefer allowing notice everywhere.